src/pages/{org}/{repo}/projects/{project_number}/issues/new.jsx

```diff
@@ -132,7 +143,9 @@ export default function NewIssuePage() {
   /**
    * @type {import("..").Store<import("..").StoreData>}
    */
-  const store = createStore(currentPath);
+
+  const store =
+    process.env.NODE_ENV === "test" ? DEFAULT_STORE : createStore(currentPath);
```

Comment:
⚠️

Reply:
We will replace this workaround with exposing the store API from the OctokitProvider in a follow up PR